sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -101,13 +101,37 @@
 if full_repo_configs_module is not None:
     try:
         module = importlib.import_module(full_repo_configs_module)
-        AVAILABLE_ONLINE_STORES = getattr(module, "AVAILABLE_ONLINE_STORES")
-        AVAILABLE_OFFLINE_STORES = getattr(module, "AVAILABLE_OFFLINE_STORES")
-    except Exception as e:
+    except ImportError as e:
         raise FeastModuleImportError(
             "FULL_REPO_CONFIGS", full_repo_configs_module
```

Comment:
I think the order of these params is flipped?

Reply:
Fixed